resources/js/components/LogIn.vue

```diff
@@ -1,24 +1,24 @@
 <template>
     <div>
-        <nav class="navbar navbar-dark bg-primary justify-content-center">
+        <nav class="npy-5 text-white bg-primary justify-content-center pt-5 pb-5">
             <h1 class="text-center">Badges</h1>
         </nav>
         <div class="container">
             <div class="row justify-content-center">
                 <form @submit.prevent="login" class="panel">
```

Comment:
Why does the form have a class of "panel"? It is not achieving anything, so please remove.